bin/run-test.sh

```diff
@@ -32,6 +32,14 @@ function run {
 # Run test/coverage
 run build/test
```

Comment:
When we run the test here, we should make sure `GCLOUD_USE_INSPECTOR` is not set.  That is, the `GCLOUD_USE_INSPECTOR` environment variable could be set globally to true for someone that is testing the protocol.

Then when this script is run, the tests are actually being run with `GCLOUD_USE_INSPECTOR=true` twice.

We should also make sure that whatever changes we make in the script the script don't leak outside the script.  That is, if `GCLOUD_USE_INSPECTOR` has been set globally to `false`, we need to make sure it's value hasn't changed to `true` after this script is run.

Reply:
I will modify the test script to reflect this.
Before running the test, I will cache GCLOUD_USE_INSPECTOR
inspector_flag=GCLOUD_USE_INSPECTOR
GCLOUD_USE_INSPECTOR=false
and after run the test, set it back.
GCLOUD_USE_INSPECTOR=inspector_flag

Reply:
Checking back, setting GCLOUD_USE_INSPECTOR won't affect environmental variable globally.

Reply:
@gaofanmichael Just for the sake of safety, can you make this line `GCLOUD_USE_INSPECTOR= run build/test` to make sure GCLOUD_USE_INSPECTOR is unset

Reply:
We need to either do `GCLOUD_USE_INSPECTOR=` to unset the variable or `unset GCLOUD_USE_INSPECTOR` to make sure the first test doesn't use inspector.